File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ## edge cases
        if len(nums) == 0:
            return [0, 0]

        ## calculate the formula
        ## nums[i] = target - nums[j]
        ## check if nums[i] is present in target - nums[j]
        for i in range(len(nums)):
            findJ = target - nums[i]

            for j in range(len(nums)):
                if findJ == nums[j] and i != j:
                    return [i, j]


        ##return output
        return [0, 0]

        

File: Data Structures & Algorithms/remove-element/submission-1.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        // edge cases

        int index =0;

        foreach(var num in nums) {
            if(num != val) {
                nums[index] = num;
                index++;
            }
        }

        return index;
    }
}